discord/interactions.py

```diff
@@ -503,6 +504,44 @@ async def delete_original_response(self) -> None:
             proxy_auth=http.proxy_auth,
         )
 
+    async def translate(
+        self, string: Union[str, locale_str], *, locale: Locale = MISSING, data: Optional[Any] = MISSING
+    ) -> Optional[Any]:
```

Comment:
```suggestion
    ) -> Optional[str]:
```
Translator.translate() returns `Optional[str]`, so shouldn't this also return that?